cpp/src/umap/fuzzy_simpl_set/naive.cuh

```diff
@@ -342,11 +342,11 @@ void launcher(int n,
                                                                               n_neighbors);
   RAFT_CUDA_TRY(cudaPeekAtLastError());
 
-  if (ML::default_logger().should_log(ML::level_enum::debug)) {
-    CUML_LOG_DEBUG("Compute Membership Strength");
+  if (ML::default_logger().should_log(ML::level_enum::trace)) {
+    CUML_LOG_TRACE("Compute Membership Strength");
```

Comment:
I think we should keep these at DEBUG level but put any actual arrays at trace level.

Reply:
Addressed in c6b6e362e768c4173fd67326f67d5b2254e38f16 .